src/components/EditCoursePage/EditCourseForm.jsx

```diff
@@ -216,6 +234,19 @@ export class BaseEditCourseForm extends React.Component {
 
     const disabled = courseInReview || !editable;
 
+    const cancelButton = (
+        <button
+          onClick={() => {
+            this.setCollapsible(false);
+            this.props.reset();
```

Comment:
Instead of calling `this.props.reset()`, you should pull `reset` out when we are deconstructing props.